config/settings/umkc.py

```diff
@@ -26,3 +26,7 @@
 
 OSLER_ABOUT_NAME = "Help"
 OSLER_ABOUT_URL = "http://www.sojournerhealthclinic.com/?page_id=1755"
+
+# Default Encounter Status
+OSLER_DEFAULT_ACTIVE_STATUS = ('Present', True)
```

Comment:
I spoke with Srivats, and these were the default statuses he wanted

Reply:
👍